### PR TITLE
Configure GitHub Actions to build and publish wheels

### PR DESCRIPTION
**Pull request by messense**
Example run: https://github.com/messense/ruuid/actions/runs/580915752

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+      - uses: actions/download-artifact@v2
+        with:
+          name: wheels
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Publish to PyPi
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
```
Comment (messense) [inline .github/workflows/CI.yml:185]: secrets needs to be configured to be able to publish to pypi.

File: Readme.md [hunk 2]
```diff
@@ -1,39 +0,0 @@
-## ruuid
```
Comment (messense) [inline Readme.md:1]: This file is duplicated with `README.md` and causes issue on macOS.

**Comment by rahulunair** [comment]
Thank you for the patch @messense

File: Readme.md (hunk 2 above)
Comment (rahulunair) [reply Readme.md:1]: good catch :)

File: .github/workflows/CI.yml [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+    - name: Upload wheels
+      uses: actions/upload-artifact@v2
+      with:
+        name: wheels
+        path: dist
+
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    if: "startsWith(github.ref, 'refs/tags/')"
```
Comment (messense) [inline .github/workflows/CI.yml:173]: Note that this will only run when git tag is pushed.

**Pull request merged**